BUILD.gn

```diff
@@ -287,9 +287,13 @@ static_library("electron_lib") {
     "//third_party/blink/renderer",
   ]
 
-  defines = []
+  defines = [ "V8_DEPRECATION_WARNINGS" ]
   libs = []
 
+  if (is_linux) {
+    defines += [ "GDK_DISABLE_DEPRECATION_WARNINGS" ]
```

Comment:
Is it necessary to disable the deprecation warnings ?

Reply:
We would have to reimplement the GTK dialog to fix the last warning